app/views/ProjectEdit/Users/UserGroupList/AddUserGroupModal/NewUsergroupSelectInput/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useMemo } from 'react';
```

Comment:
Is UsergroupSelectInput being used somewhere else? If not, lets make this the default one, and remove the work new.